docker/dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the rust image as the base image
+FROM rust:latest
+
+# Install the dependencies required by Rust-TEOS
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install libffi-dev libssl-dev pkg-config libleveldb-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Set the working directory to /srv
+WORKDIR /srv
+
+# Copy the source code into the container
+COPY . /srv/rust-teos
+
+# Build the application
+WORKDIR /srv/rust-teos
+
+#Getting [ error: 'rustfmt' is not installed for the toolchain '1.67.1-aarch64-unknown-linux-gnu'] so added a line:
+RUN rustup component add rustfmt --toolchain 1.67.1-aarch64-unknown-linux-gnu  
```

Comment:
ummm, aarch64?
Isn't this docker container an amd64 arch?

Reply:
Dockerfile I provided does not specify the platform architecture for the Docker image. So it uses my system architecture which is aarch64/ARM64. Should I make one for amd64 too?

Reply:
oh well, now that makes sense.
I think if you omit the toolchain option you will and just do `add rustfmt` it will download the right one for your system arch.
The thing is, most servers are amd64 so you wanna support that, but aarch64 is of course a plus to have.

Reply:
Ohh yesss.. Got that. Thanks 😁